web/src/containers/OpenOrders/index.tsx

```diff
@@ -1,180 +1,164 @@
 import classnames from 'classnames';
-import * as React from 'react';
-import { Spinner } from 'react-bootstrap';
-import { FormattedMessage, injectIntl } from 'react-intl';
-import { connect, MapDispatchToPropsFunction } from 'react-redux';
-import { IntlProps } from '../../';
+import React, { useCallback, useMemo, useState } from 'react';
+import { Form, Spinner } from 'react-bootstrap';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { useOpenOrdersFetch } from 'src/hooks';
 import { CloseIcon } from '../../assets/images/CloseIcon';
 import { Decimal, OpenOrders } from '../../components';
 import { localeDate, setTradeColor } from '../../helpers';
 import {
-    Market,
     openOrdersCancelFetch,
     ordersCancelAllFetch,
-    RootState,
-    selectCancelOpenOrdersFetching,
     selectCurrentMarket,
+    selectMarkets,
     selectOpenOrdersFetching,
     selectOpenOrdersList,
-    selectUserLoggedIn,
-    userOpenOrdersFetch,
 } from '../../modules';
 import { OrderCommon } from '../../modules/types';
 
-interface ReduxProps {
-    currentMarket: Market | undefined;
-    list: OrderCommon[];
-    fetching: boolean;
-    cancelFetching: boolean;
-    userLoggedIn: boolean;
-}
-
-interface DispatchProps {
-    userOpenOrdersFetch: typeof userOpenOrdersFetch;
-    openOrdersCancelFetch: typeof openOrdersCancelFetch;
-    ordersCancelAll: typeof ordersCancelAllFetch;
-}
-
-type Props = ReduxProps & DispatchProps & IntlProps;
-
-export class OpenOrdersContainer extends React.Component<Props> {
-    public componentDidMount() {
-        const { currentMarket, userLoggedIn } = this.props;
-        if (userLoggedIn && currentMarket) {
-            this.props.userOpenOrdersFetch({ market: currentMarket });
+const getTriggerSign = (ordType: string, ordSide: string) => {
+    if (ordType.includes('stop')) {
+        if (ordSide === 'buy') {
+            return '≥';
+        } else {
+            return '≤';
         }
     }
 
-    public componentWillReceiveProps(next: Props) {
-        const { userLoggedIn, currentMarket } = next;
-        const { userLoggedIn: prevUserLoggedIn, currentMarket: prevCurrentMarket } = this.props;
-
-        if (!prevUserLoggedIn && userLoggedIn && currentMarket) {
-            this.props.userOpenOrdersFetch({ market: currentMarket });
-        } else if (userLoggedIn && currentMarket && prevCurrentMarket !== currentMarket) {
-            this.props.userOpenOrdersFetch({ market: currentMarket });
+    if (ordType.includes('take')) {
+        if (ordSide === 'buy') {
+            return '≤';
+        } else {
+            return '≥';
         }
     }
+}
```

Comment:
maybe better to create helpers.ts inside web/src/containers/OpenOrders folder and move this function there?